remoteworkitem/remoteworkitem_test.go

```diff
@@ -43,14 +59,15 @@ func TestWorkItemMapping(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	if workItem.Fields["system.title"] == nil {
+	if !assert.NotNil(t, workItem.Fields["system.title"]) {
```

Comment:
But you can ass the error message as a third parameter to `assert.NotNil(t, value, msg)`.